### PR TITLE
Admin dasboard demo

### PR DESCRIPTION
**Pull request by LaserHuy**

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -175,3 +176,26 @@ INSERT INTO `Student_Reviews` (`student_id`, `testimonial`) VALUES
 (4, 'Je n’ai jamais porté une grande importance aux diplômes, mais plutôt à mes compétences, à ce que je suis capable de faire concrètement. Je cherchais à allier la partie technique et le côté relationnel et fonctionnel dans la suite de ma carrière'),
 (5, 'C’était très intense, j’ai eu une promotion en or et beaucoup d’entraide et ça ça a été très important.'),
 (6, 'Peu importe ce que l’on a fait avant, c’est toujours une force, et non une faiblesse.');
+
+
+--
+-- Contenu de la table `Course`
+--
+
+INSERT INTO `Course` (`id` , `name` , `description` , `capacity` , `location`, `date`, `duration` , `degree` , `financing_supported` , `discipline_id`, `url_image`) VALUES
```
Comment (Benja-san) [inline database.sql:185]: remove id from this sql request (and values associated) You ain't gonna need it since ID is auto generated and auto incremented

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: database.sql (hunk 1 above)
Comment (LaserHuy) [reply database.sql:185]: remove succesfully

**Review by Benja-san — APPROVED** [review]

**Review by assma8jaballah — APPROVED** [review]

File: public/assets/css/admin/main-admin.css [hunk 2]
```diff
@@ -0,0 +1,354 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;400;500;600;700;800;900;1000&family=Roboto:wght@300;400;500;700&display=swap");
+
+.main-admin {
+    margin: 0;
+    font-family: "Nunito", sans-serif;
```
[the remark this answers is not in the document]
Comment (LaserHuy) [reply public/assets/css/admin/main-admin.css:5]: Yeah I change just font for admin page

[the remark this answers is not in the document]
Comment (LaserHuy) [reply public/assets/css/admin/main-admin.css:5]: only this font will apply for admin page

**Pull request merged**